raw/Makefile

```diff
@@ -106,123 +106,14 @@ config.h:
 libevercrypt.$(SO): config.h $(OBJS)
 	$(CC) $(CFLAGS) -shared -o $@ $(filter-out %.h,$^) $(LDFLAGS)
 
+libevercrypt.a: config.h $(OBJS)
+	ar rc $@ $+
```

Comment:
This passes `config.h` to `ar`, I don't think it should do that.

Reply:
That explains the `filter-out` — good point, I've applied the same fix as above.